tests/network-tests/src/constantinople/tests/impl/membershipCreation.ts

```diff
@@ -15,15 +16,16 @@ export function membershipTest(nKeyPairs: KeyringPair[]) {
   const paidTerms: number = +process.env.MEMBERSHIP_PAID_TERMS!;
```

Comment:
For all the functions inside the `impl` folder which are now the reusable unit, it might be better to have these scenario parameters be passed as arguments and the caller would read them from `process.env`

`registerJoystreamTypes()` is also called in each test setup, and I think the caller of the test only needs to do that one time, and might as well also create the api instance and pass that as argument as well.

Reply:
Scenario parameters now read from environment in scenarios and are passed to the implementation as parameters.

Type registration and api instance are still test-specific. I've run into issues while trying move them up to the scenario level. Let me know if we should discuss it.